examples/plot_bcic_iv_2a_moabb_cropped.py

```diff
@@ -92,7 +93,7 @@
 
 preprocessors = [
     Preprocessor('pick_types', eeg=True, meg=False, stim=False),  # Keep EEG sensors
-    Preprocessor(scale, factor=1e6, apply_on_array=True),  # Convert from V to uV
+    Preprocessor(lambda data, factor: multiply(data, factor), factor=1e6),  # Convert from V to uV
```

Comment:
```suggestion
    Preprocessor(partial(multiply, 1e6)),  # Convert from V to uV
```
should also work? Maybe it is nicer?
Also wondered if we could also forward *args to the function be applied and this way write something like
```
Preprocessor(multiply,2)
```
What do you think @bruAristimunha ?

Reply:
I like the first option, using the partial, but this is definitely not a strong opinion.

Reply:
whichever option you choose, I'm happy

Reply:
What do you think @agramfort?

Reply:
No strong feeling. Partial is more pythonic but maybe harder for new comers. Now our custom API is not standard either. So as you prefer

Reply:
Let's go for simple option, okay @robintibor?